File: 07_ServerlessFargatePipeline/typescript/lib/serverless-stage.ts
import { ServerlessStack } from './serverless-stack';
import { Stage, StageProps } from 'aws-cdk-lib';
import { Construct } from 'constructs';

export class ServerlessStage extends Stage {
    constructor(scope: Construct, id: string, props?: StageProps) {
        super(scope, id, props);

        new ServerlessStack(this, 'ServerlessApp');
    }
}

File: 07_ServerlessFargatePipeline/python/ServerlessFargatePipeline/serverless_stage.py
from constructs import Construct
from aws_cdk import (
    Stage
)
from .serverless_stack import ServerlessStack

class ServerlessStage(Stage):

    def __init__(self, scope: Construct, id: str, **kwargs):
        super().__init__(scope, id, **kwargs)

        service = ServerlessStack(self, 'ServerlessStack')